packages/x-date-pickers-pro/src/DateRangeCalendar/useDragRange.ts

```diff
@@ -152,12 +152,6 @@ const useDragRangeEvents = <TDate>({
     }
 
     setRangeDragDay(newDate);
-    setIsDragging(true);
```

Comment:
The underlying issue seemed to stem from the fact that `onTouchStart` was being triggered after taping on a date, thus initializing the touch sequence and ending it with a "confirmation" of selection on `touchEnd`, which used to result in a double selection of the same date.

Moving the `setIsDragging` to `onTouchMove` seems logical as it better resembles how the `onDragStart` on desktop behaves. 🤔 
Doing so fixes the issue of double selection and the overall behavior works as expected.

Reply:
I did try that as well, since it made the most sense. But if we do that it results in a weird flash of a previous state. But it could very well be that it was due to some other changes i did. Will have a look and try to fix this week! 💯

Reply:
Could you maybe record a video of the problem you are referring to? 🤔 
As far as I've tested, I did not see any strange visual behavior, but I could be very well missing something. 🙈